### PR TITLE
Reorganise documentation

### LINKED ISSUE #275: Make the docs correctly represent the submodules that make up EpiAware and surface the Public and private APIs
As the title.

### PR DESCRIPTION
**Pull request by seabbs**
This PR closes #275 and # by reorganising the documentation. Note that many of the files are placeholders and need content (and also issues to drive content creation).

> [!WARNING]
> I haven't tested this locally so it may ping off the CI a few times as I update it.

### CODE REVIEW

**Comment by seabbs** [comment]
@SamuelBrand1 what do you think of this so far? 

Any thoughts as to why the `build` function is having problems? As far as I can tell nothing has changed here that should be causing the pathing error it is throwing and yet...

**Comment by SamuelBrand1** [comment]
Looks like you need to move the underlying data? Missing dir/file is causing the documenter CI fail.

File: EpiAware/docs/src/getting-started/tutorials/intro.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+The diagram below shows the relationship between the modules in the package for a typical workflow.
```
Comment (SamuelBrand1) [inline EpiAware/docs/src/getting-started/tutorials/intro.md:1]: I think this mermaid diagram needs updated? Maybe make an issue for that.

**Review by SamuelBrand1 — DISMISSED** [review]
I really like the new structure for the docs, and it seems a lot more user readable.

File: EpiAware/docs/src/getting-started/tutorials/intro.md (hunk 1 above)
Comment (seabbs) [reply EpiAware/docs/src/getting-started/tutorials/intro.md:1]: yup it does. Once this is merged going to make a wave of issues for all the doc gaps

**Comment by seabbs** [comment]
I think this should now build. Note I did some reorganisation vs before. The issue was in the @contents call. It looks like using the side bar panels has some confusing bugs that are hard to pin down. The main issue was that even commenting them out didn't do away with the. bug

**Review by SamuelBrand1 — APPROVED** [review]
LGTM. I like the rename to explainers

**Review by SamuelBrand1 — APPROVED** [review]

**Pull request merged**